src/App.js

```diff
@@ -1,14 +1,23 @@
-import Todo from './components/Todo.js';
+import FeedPage from "./pages/Feed.js";
+// import NewPostPage from "./pages/NewPost.js";
```

Comment:
In general you should try and avoid committing commented out code, in general you can try to stash it, commit it on a different branch uncommented if it is still useful or sometimes a comment to explain why it is necessary is also reasonable

Reply:
Yeah I need to put more effort into going through and removing them before a PR, because I don't think any of them are useful